index.html

```diff
@@ -16,32 +17,171 @@
+          product , feature or website Look through some
+          ce! If you like what you see and have a project you need code
+          d, don’t hestiate to contact me.</p>
+        <p class="connect">LETS CONNECT</p>
+      </section>
+      <section class="icons">
+        <ul>
+          <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+          <li><a href="#"><img src="images/in.png" alt="in"></a></li>
+          <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
+          <li><a href="#"><img src="images/cat.png" alt="cat"></a></li>
+          <li><a href="#"><img src="images/fingers.png" alt="fingers"></a></li>
+        </ul>
+      </section>
+
     </section>
-    <section class="icons">
-      <ul>
-        <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
-        <li><a href="#"><img src="images/in.png" alt="in"></a></li>
-        <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
-        <li><a href="#"><img src="images/cat.png" alt="cat"></a></li>
-        <li><a href="#"><img src="images/fingers.png" alt="fingers"></a></li>
+    <section class="tonic2">
+      <img src="images/tonic.png" alt="tonic">
+      <h1 class="tnc">Tonic</h1>
+      <ul class="ul2">
+        <li class="can">CANOPY</li>
+        <li class="li2">Back End Dev</li>
+        <li class="li2">2015</li>
       </ul>
+      <p>A daily selection of privately
+        personalized reads; no accounts or sign-ups
+        required.</p>
+      <ul class="ul3">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+
+      <a href="#" class="sp">See Project</a>
+
     </section>
+    <section class="tonic2">
+      <img src="images/multpst2.png" alt="tonic">
+      <h1 class="tnc">Multi-Post <br>
+        Stories</h1>
+      <ul class="ul2">
+        <li class="can">CANOPY</li>
+        <li class="li2">Back End Dev</li>
+        <li class="li2">2015</li>
+      </ul>
+      <p>A daily selection of privately
+        personalized reads; no accounts or sign-ups
+        required.</p>
+      <ul class="ul3">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+
+      <a href="#" class="sp">See Project</a>
+
+    </section>
+    <section class="tonic2">
+      <img src="images/tonic3.png" alt="tonic">
+      <h1 class="tnc">Tonic</h1>
+      <ul class="ul2">
+        <li class="can">CANOPY</li>
+        <li class="li2">Back End Dev</li>
+        <li class="li2">2015</li>
+      </ul>
+      <p>A daily selection of privately
+        personalized reads; no accounts or sign-ups
+        required.</p>
+      <ul class="ul3">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+
+      <a href="#" class="sp">See Project</a>
 
-  </section>
+    </section>
+    <section class="tonic2">
+      <img src="images/multipost.png" alt="tonic">
+      <h1 class="tnc">Multi-Post <br>
```

Comment:
Kindly change it to h2 as you have already used h1 above

Reply:
- Well addressed.